### PR TITLE
chore: prefer usize::MAX to std::usize::MAX

### PR DESCRIPTION
**Pull request by tisonkun**
Per https://github.com/smol-rs/polling/pull/140#issue-1859612645

Hopefully, we can move `Event`s free. Open for comments :D

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -136,6 +135,9 @@ pub struct Event {
     extra: sys::EventExtra,
 }
 
+unsafe impl Send for Event {}
+unsafe impl Sync for Event {}
```
Comment (notgull) [inline src/lib.rs:139]: I don't like the additional `unsafe` here. Can we just add a test to make sure the `Event` struct is `Send` and `Sync`?

Comment (tisonkun) [reply src/lib.rs:139]: > add a test

Do you mean that we ensure the auto trait happens by testing? What's the difference to this patch that let the compiler do the "test"?

These two lines are automatically done if it meets the auto trait conditions. We declare them explicitly for testing here.

Comment (notgull) [reply src/lib.rs:139]: Like this: https://github.com/smol-rs/async-executor/blob/2cfb6e4ed03c7eaf6c5fb2d3170ac870ba0a9b9d/src/lib.rs#L1065

Comment (tisonkun) [reply src/lib.rs:139]: OK. I'm pushing a commit now - this seems a style preference :D

Comment (tisonkun) [reply src/lib.rs:139]: We already have it in `_assert_send_and_sync`.

Then I'll convert this patch as a trivial code tidy one.

**Review by notgull — APPROVED** [review]
Thanks!

**Pull request merged**